plugins/auth-keycloak/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@amplication/plugin-auth-keycloak",
+  "version": "0.0.4",
+  "description": "Enable Keycloak authentication on a service",
+  "main": "dist/index.js",
+  "scripts": {
+    "dev": "webpack --watch",
+    "build": "webpack",
+    "prebuild": "rimraf dist",
+    "format:write": "prettier --write \"**/*.{js,ts}\"",
+    "lint:fix": "eslint './src/**/*.{ts, tsx}' --fix --c .eslintrc",
+    "lint": "eslint './src/**/*.{ts, tsx}' --c .eslintrc",
+    "test": "jest"
+  },
+  "author": "Damanpreet singh<daman.4880@gmail.com>",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@amplication/code-gen-types": "^2.0.12",
+    "@amplication/code-gen-utils": "^0.0.6",
```

Comment:
@fumblehool do you mind updating these dependencies to the latest plugin-template